scripts/policy/misc/unknown-protocols.zeek

```diff
@@ -0,0 +1,58 @@
+##! This script logs information about packet protocols that Zeek doesn't
+##! know how to process. Mostly these come from packet analysis plugins when
+##! they attempt to forward to the next analyzer, but they also can originate
+##! from non-packet analyzers.
+
+@load base/frameworks/notice
+
+module UnknownProtocol;
+
+export {
+	redef enum Log::ID += { LOG };
+
+	global log_policy: Log::PolicyHook;
+
+	type Info: record {
+		## Timestamp for when the measurement occurred.
+		ts:           time     &log;
+		##
+		analyzer:     string   &log;
+		protocol:     string   &log;
+		first_bytes:  string   &log;
```

Comment:
I think `first_bytes` is pretty descriptive. Is `protocol` intended to be more general? Otherwise, it could be renamed to `protocol_id` and become a count.

Reply:
It's a string so that it can be logged as a hex value, but I renamed it.